internal/pkg/scaffold/helm/go_mod.go

```diff
@@ -137,30 +137,29 @@ require (
 	k8s.io/cli-runtime v0.0.0-20181213153952-835b10687cb6 // indirect
 	k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
 	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a // indirect
-	k8s.io/helm v2.13.1+incompatible // indirect
+	k8s.io/helm v2.14.1 // indirect
 	k8s.io/klog v0.2.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20190603182131-db7b694dc208 // indirect
-	k8s.io/kubernetes v0.0.0-20190201210629-c6d339953bd4 // indirect
+	k8s.io/kubernetes v1.14.1 // indirect
 	k8s.io/utils v0.0.0-20190308190857-21c4ce38f2a7 // indirect
-	sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-runtime v0.2.0-beta.3
 	sigs.k8s.io/controller-tools v0.1.10
 	sigs.k8s.io/testing_frameworks v0.1.1 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787 // indirect
 )
 
-// Pinned to kubernetes-1.13.4
+// Pinned to kubernetes-1.14.1
 replace (
-	k8s.io/api => k8s.io/api v0.0.0-20190222131558-5cb15d344471
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221124651-86fb29eff628
-	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20190228174905-79427f02047f
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.0.0-20190228180923-a9e421a79326
-	k8s.io/client-go => k8s.io/client-go v0.0.0-20190228174230-b40b2a5939e4
-	k8s.io/kubernetes => k8s.io/kubernetes v1.13.4
+	k8s.io/api => k8s.io/api kubernetes-1.14.1
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver kubernetes-1.14.1
+	k8s.io/apimachinery => k8s.io/apimachinery kubernetes-1.14.1
+	k8s.io/apiserver => k8s.io/apiserver kubernetes-1.14.1
+	k8s.io/cli-runtime => k8s.io/cli-runtime kubernetes-1.14.1
+	k8s.io/client-go => k8s.io/client-go kubernetes-1.14.1
+	k8s.io/cloud-provider => k8s.io/cloud-provider kubernetes-1.14.1
+	k8s.io/kubernetes => k8s.io/kubernetes v1.14.1
 )
-
-replace github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
```

Comment:
This isn't needed anymore?
For reference this was added here: https://github.com/operator-framework/operator-sdk/commit/ed575591df4e924ebc99aed6c62ca113a325fef4

Reply:
I noticed it wasn't in Helm's [`gopkgtoml.go`](https://github.com/operator-framework/operator-sdk/blob/e0342c4/internal/pkg/scaffold/helm/gopkgtoml.go) so I removed it.

Reply:
From a build on master, I ran:
```console
operator-sdk new nginx-operator --type=helm --api-version=example.com/v1 --kind=Nginx
cd nginx-operator
operator-sdk migrate
git init && git add . && git commit -m "initial commit"
go mod tidy
```

And it ends up in `go.mod` as an indirect dependency:
```console
$ go mod why -m github.com/coreos/prometheus-operator
# github.com/coreos/prometheus-operator
github.com/test-operators/nginx-operator/cmd/manager
github.com/operator-framework/operator-sdk/pkg/helm
github.com/operator-framework/operator-sdk/pkg/metrics
github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1
```

There are quite a few other changes to `go.mod` that are probably worth evaluating in the git diff after running that last `go mod tidy` command.

Reply:
Ok so we can add this to `require` instead of `replace`.

Reply:
I _think_ that's right. I think it's an indirect dependency, which means `go` might change it on you, in which case, we would need both.

Reply:
Should it be added to `gopkgtoml.go` too?

Reply:
So dep prunes unused packages, and that package is currently unused, so I don't _think_ it's required in `gopkgtoml.go`. I know @LiliC has https://github.com/operator-framework/operator-sdk/pull/1601 open, which may end up pulling it in.